test/unittest/helper.go

```diff
@@ -37,7 +41,7 @@ func GetVpcDefaultPlans(t *testing.T) *terraform.PlanStruct {
 }
 
 func GenerateTFOptions(variables map[string]interface{}, t *testing.T, module string) *terraform.Options {
-	exampleFolder := testStructure.CopyTerraformFolderToTemp(t, "../../pkg/modules/AWS", "/"+module)
+	exampleFolder := testStructure.CopyTerraformFolderToTemp(t, ProductModulePath, "/"+module)
```

Comment:
This was the main reason why I had to change the structure `testStructure.CopyTerraformFolderToTemp` didn't work with the `pkg/products/bamboo` as it was searching for `modules folder`

Reply:
This looks a bit weird but probably this is based on terratest design.